default.nix

```diff
@@ -98,8 +98,11 @@ rec {
     '';
   };
 
-  tests = import ./nix/tests/vm-basic.nix {
-    inherit pkgs;
-    wstModule = module;
-  };
+  inherit
+    (import ./nix/tests/vm-basic.nix {
+      inherit pkgs;
+      wstModule = module;
+    })
+    vm-basic
```

Comment:
This now makes it impossible to do `nix-build -A tests` to build all NixOS tests, right?

Reply:
You would run `nix-build` to run all tests instead, so it's actually easier now. Those formatting tests are also important and shouldn't be missed.

Reply:
This is now possible again.